os/board/rtl8721csm/src/component/common/api/example_eap.c

```diff
@@ -0,0 +1,238 @@
+#include <platform_opts.h>
+#include <example_eap.h>
+#include "osdep_service.h"
+#include <platform/platform_stdlib.h>
+#include <rtl8721d_crypto.h>
+#include <rtk_wifi_utils.h>
+
+#if CONFIG_EXAMPLE_EAP

```

Comment:
Could you let me where this definition is? Could you confirm that `#if` is right? not `#ifdef`?

Reply:
Dear @sunghan-chang ,
Sure, this macro is defined in \os\board\rtl8721csm\src\project\realtek_amebad_va0_example\inc\inc_hp\platform_opts.h as: #define CONFIG_EXAMPLE_EAP	0